docs/admin-manual/tutorials/setup-liquidsoap-aac-streaming.md

```diff
@@ -0,0 +1,48 @@
+---
+title: How to update liquidsoap to support AAC streaming
+---
+
+This tutorials walks you though the steps required to replace the liquidsoap package with a version that supports AAC streaming.
```

Comment:
Add a warning at the top of this page talking about the lack of security support and the dangers of following this to get AAC working.